boot/boot.js

```diff
@@ -1444,6 +1444,7 @@ $tw.loadTiddlersBrowser = function() {
 	for(var t=0; t<containerIds.length; t++) {
 		$tw.wiki.addTiddlers($tw.wiki.deserializeTiddlers("(DOM)",document.getElementById(containerIds[t])));
 	}
+	callback();
```

Comment:
@danielo515 :+1:

I'm confused what this is trying to do. A synchronous function can exists in an async environment. It is the reverse that is problematic. In this PR it looks like we are pseudo-async (pretending to be async without any benefit) in which case we don't need any callbacks.

What is a callback API providing that the non-callback API can not do (in this example?)